### ISSUE TITLE
New Term Request - Parosmia

### ISSUE THREAD

**Issue by danteXmachina** [obophenotype/human-phenotype-ontology#7050]
**Preferred term label:** Parosmia

**Synonyms** Olfaction Disorder
Smell Disorders
Smell Disorder
Cacosmia
Cacosmias
Dysosmia
Dysosmias
Paraosmia
Paraosmias
Smell Dysfunction
Dysfunction, Smell
Olfactory Impairment
Impairment, Olfactory

References: https://www.ncbi.nlm.nih.gov/mesh/?term=parosmia, MeSH Unique ID: D000857 

**Definition (free text, please give PubMed ID)** Inability of the brain to properly identify an odors natural smell. Distinct from anosmia (PMID: 32741085) and hyposmia (PMID:31356735) , parosmia has been identified as a post-infectious consequence of olfactory nerve injury by respiratory tract pathogens (PMID: 33210732),  including SARS-CoV-2 (PMID: 32605712). Parosmia can arise by incomplete or aberrant regeneration of the injured olfactory nerves (PMID:31356735), or by defects in the central processing hubs of olfactory input independently of hyposmia (PMID: 31238309). Neurodegenerative causes of parosmia include network degeneration extending from the olfactory nerve to its projections in hippocampi and cortical grey matter (PMID: 21241781). As a symptom, parosmia is coincident with several neurological disorders including dementias (PMID: 33240192).

See also: https://www.ncbi.nlm.nih.gov/medgen/307133

**Parent term (use hpo.jax.org/app)** 

Abnormality of the sense of smell

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

Upper respiratory tract infections PMID:31356735
COVID-19 PMID: 32605712
Alzheimer's disease PMID: 33240192

Please not that Orphanet and OMIM have not catalogued anosmia. 

**Your nano-attribution (ORCID)**

ORCID: 0000-0002-0413-4028 , Google Scholar: https://scholar.google.com/citations?user=bYO5hS4AAAAJ&hl=en

**Comment by pnrobinson** [comment]
@danteXmachina  it is unclear to me exactly what the clinical difference is. In the article cited above, the patient is described as having complete anosmia due to COVID-19 infection, but the authors then go on to call it parosmia, but it seems they are using it synonymously. 
We have been working on the long COVID phenotype (https://www.medrxiv.org/content/10.1101/2021.06.23.21259416v1), and from this I believe that COVID can be characterized by anosmia, hyposmia, and phantosmia -- is there anything else?

**Comment by danteXmachina** [comment]
@pnrobinson anosmia and hyposmia refer to complete or partial loss of the olfactory perception (i.e. quantitative disorders); phantosmia is olfactory perception without stimuli. Parosmia is altered olfactory perception in the presence of known stimuli, such as the smell of coffee; along with phantosmia, it is a qualitative disorder. It is clinically and phenomenologically distinct, and although bundled in with anosmia, it is separate clinical entity which involves cortical circuits, not just receptors. I've included two somewhat better references on COVID-19 and parosmia.

https://pmj.bmj.com/content/early/2021/03/31/postgradmedj-2021-139855
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8141364/

**Comment by danteXmachina** [comment]
@pnrobinson PS.: if it would help, I could summarize studies of parosmia (specifically) in COVID-19 as a mini review in a subsequent comment, and its associations with neurological complications in the LongCOVID spectrum. It's easy to do, and more or less our field work and its theoretical framework put into use (https://www.alz.org/aaic/downloads2021/COVID-19_and_Long-Term_Cognitive_Dysfunction.pdf)

**Comment by pnrobinson** [comment]
@danteXmachina  -- I have added the term with the following
definition: Misperception of existing odor, that is, distorted or altered olfactory perception in the presence of known stimuli.
comment: Parosmia should be distinguished from anosmia (lack of sense of smell), hyposmia (reduced sense of smell), and phantosmia (olfactory perception without a stimulus).

Please contact me by email about the long COVID analysis! Here are my contact data https://robinsongroup.github.io/contact.html

**Comment by danteXmachina** [comment]
Thank you  for the immediate response and the accurate representation of the term! I have sent an email at the indicated address.

Best,

George
